contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -67,6 +67,15 @@ contract BasicAMBErc677ToErc677 is
         return isInitialized();
     }
 
+    function getReceiver(address _from, bytes _data) internal view returns (address recipient) {
+        recipient = _from;
+        if (_data.length == 20) {
+            recipient = Bytes.bytesToAddress(_data);
+        }
+        require(recipient != address(0));
+        require(recipient != mediatorContractOnOtherSide());
```

Comment:
I thinks they should be within `if` since definitely will be `true` if `recipient` equals `_from`

Reply:
Thanks, fixed https://github.com/poanetwork/poa-bridge-contracts/pull/301/commits/6601874ea3b95ede5b545c384217ae547cc98347